mediation-old/src/main/java/com/wso2telco/dep/mediator/util/ValidationUtils.java

```diff
@@ -30,13 +35,37 @@
 
     static Log log = LogFactory.getLog(ValidationUtils.class);
 
-    public static void compareMsisdns(String urlmsisdn, String payloadMsisdn){
+    /**
+	 * This method extracts userId from payload and resource url and passed to
+	 * validate whether they are same
+	 */
```

Comment:
I think it's time to move this method to a Util class. We are duplicating the functionality across two classes. The reason why we had to write it in AmountChargeHandler is because that was the only class doing MSISDN comparison at that time.

Reply:
Addressed the review comment